js/content/store.js

```diff
@@ -912,6 +913,44 @@ class AppPageClass extends StorePageClass {
         });
     }
 
+    addFullscreenScreenshotView() {
+        function es_toggleFullScreen(event) {
```

Comment:
I don't think we ever prefixed functions with `es_`. You may want to remove that for consistency sake.